src/main/java/hudson/plugins/plot/Plot.java

```diff
@@ -149,25 +149,25 @@ public class Plot implements Comparable<Plot> {
      * Number of builds back to show on this plot from url.
      */
     @SuppressWarnings("visibilitymodifier")
-    public transient String urlNumBuilds;
+    private transient String urlNumBuilds;
```

Comment:
I've made the mistake in the past of changing the visibility of a public variable and then breaking the serialization of the object or the dependencies that other plugins had on the object.

It seems simpler to exclude the spotbugs warning in these types of cases rather than risk incompatibility for users.

Reply:
Got it, thanks.
I have proposed #132 then.